web/src/jobs/sessions.rs

```diff
@@ -0,0 +1,28 @@
+use cja::{app_state::AppState as _, jobs::Job};
+use miette::IntoDiagnostic;
+
+use crate::AppState;
+
+#[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
+pub struct Cleanup;
+
+#[async_trait::async_trait]
+impl Job<AppState> for Cleanup {
+    const NAME: &'static str = "sessions::cleanup";
+
+    async fn run(&self, app_state: AppState) -> miette::Result<()> {
+        let db = app_state.db();
+
+        sqlx::query!(
+            r#"
+            DELETE FROM sessions
+            WHERE expires_at < now() - interval '1 day'
+            "#,
+        )
+        .execute(db)
+        .await
+        .into_diagnostic()?;
+
+        Ok(())
+    }
```

Comment:
This is how we write jobs! We have a struct call `Cleanup` and then we implement the `Job<AppState>` trait

This required we implement on method called `run` where we want to logic of our job to live!

Here we simply delete old sessions!

Reply:
`Job <AppState>` is imported from the `cja` crate?

Reply:
If so, where can I find the code for `App State` trait?